setup.cfg

```diff
@@ -59,3 +59,4 @@ exclude_lines =
     pragma: no ?cover
     pragma ${PLATFORM_SYSTEM}: no cover
     pragma ${OS_NAME}: no cover
+    pragma Python [^: ]*${PYTHON_VERSION}[^: ]*: no cover
```

Comment:
I've expanded it a bit using 0-9 as it is one more character and much easier to read for newbies. ;-)